bezier-rs/docs/interactive-docs/wasm/src/lib.rs

```diff
@@ -1,25 +1,40 @@
 pub mod subpath;
 mod svg_drawing;
 
-use bezier_rs::{Bezier, ProjectionOptions};
+use bezier_rs::{ArcsOptions, Bezier, MaximizeArcs, ProjectionOptions};
 use glam::DVec2;
 use serde::{Deserialize, Serialize};
 use wasm_bindgen::prelude::*;
 
+#[derive(Serialize, Deserialize)]
+struct CircleSector {
+	center: Point,
+	radius: f64,
+	start_angle: f64,
+	end_angle: f64,
+}
+
 #[derive(Serialize, Deserialize)]
 struct Point {
 	x: f64,
 	y: f64,
 }
 
+#[wasm_bindgen]
+pub enum WasmMaximizeArcs {
+	Automatic, // 0
+	On,        // 1
+	Off,       // 2
```

Comment:
Please use the "unit" feature to try and convey to the user what each of these mean. So perhaps the unit can be something like:
```
(0: auto, 1: on, 2: off)
```